### PR TITLE
Update epoch node configuration

### PR DESCRIPTION
**Pull request by dincho**
- change configuration format to YAML
- enable persistance
- autostart miner

Because of bug in the chain.db_path it's always set to /.chain.

### CODE REVIEW

File: ci [hunk 1]
```diff
@@ -59,7 +59,7 @@ case "${1:?}"-"${2:?}" in
         BuiltPackagePath="${BuildDir:?}"/_build/prod/rel/epoch/epoch-"${Version:?}".tar.gz
         OsLabel=$(package_os_label "${TRAVIS_OS_NAME:?}")
         FinalPackagePath="${BuildDir:?}"/_build/prod/rel/epoch/epoch-"${Version:?}"-"${OsLabel:?}".tar.gz
-        cp -p "${BuiltPackagePath:?}" "${FinalPackagePath:?}"
+        mv "${BuiltPackagePath:?}" "${FinalPackagePath:?}"
```
Comment (lucafavatella) [inline ci]: Thanks

File: deployment/ansible/deploy.yml [hunk 2]
```diff
@@ -5,6 +5,7 @@
 
   vars:
     project_root: "{{ ansible_env.HOME }}/node"
+    chain_db_path: "/.chain"
```
Comment (lucafavatella) [inline deployment/ansible/deploy.yml]: @dincho Do I understand correctly that once the default path moves to `.` you need neither this line nor L45-L67?

I would suggest not adding this workaround, but rather ensure the related PRs get merged...

Comment (uwiger) [reply deployment/ansible/deploy.yml]: Was it really meant to be a hidden directory? Why?

**Comment by lucafavatella** [comment]
> Please remove chain db path workaround.

@dincho I understand that as long as PR https://github.com/aeternity/epoch/pull/376 is merged before this, it is safe to remove workaround.

I understand PR https://github.com/aeternity/epoch/pull/377 is not needed for this PR.

Cc @velzevur @uwiger

**Comment by lucafavatella** [comment]
@dincho PR #376 was merged so this PR can be amended removing workaround.

**Comment by dincho** [comment]
> @dincho I understand that as long as PR #376 is merged before this, it is safe to remove workaround.

Yes, I'll do. Is the daemon creates the configured directory automatically or we need to do it prior configuration?

**Comment by lucafavatella** [comment]
> Yes, I'll do. Is the daemon creates the configured directory automatically or we need to do it prior configuration?

The deamon creates the configured directory automatically.

**Comment by lucafavatella** [comment]
I see one of the two Travis CI builds passed.

I see the other Travis CI build had all required jobs passing except package job on OSX:
```
Will mine till block 50
[node1] height=20
[node2] height=2
[node3] height=20

[node1] height=20
[node2] height=2
[node3] height=20

[node1] height=20
[node2] height=2
[node3] height=20

[node1] height=21
[node2] height=2
[node3] height=21

[node1] height=21
[node2] height=2
[node3] height=21

[node1] height=21
[node2] height=2
[node3] height=21

[node1] height=23
[node2] height=2
[node3] height=23

[node1] height=23
[node2] height=2
[node3] height=23

[node1] height=24
[node2] height=2
[node3] height=24

[node1] height=25
[node2] height=2
[node3] height=25

[node1] height=25
[node2] height=2
[node3] height=25

...
```

@dincho This looks like intermittent test failure independent from this PR.

Cc @velzevur @sennui @wagerlabs

**Pull request merged**